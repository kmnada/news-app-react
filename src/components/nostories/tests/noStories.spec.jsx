import React from 'react';
import { shallow } from 'enzyme';
import NoStories from '../noStories';

describe(' NoStories component', () => {
  it('renders default component', () => {
    const wrapper = shallow(<NoStories />);
    expect(wrapper).toMatchSnapshot();
  });
});
