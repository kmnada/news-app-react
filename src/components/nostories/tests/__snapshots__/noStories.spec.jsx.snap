// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` NoStories component renders default component 1`] = `ShallowWrapper {}`;
